src/getCoinCombination.test.js

```diff
@@ -3,8 +3,68 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
+  test(`should be declared`, () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test(`should be an array`, () => {
+    expect(getCoinCombination())
+      .toBeInstanceOf(Array);
+  });
+
+  test(`should return 0 coins`, () => {
```

Comment:
Maybe add more clear explanation? For ex. "Should return [0, 0, 0, 0] when the input is 0"